litr-filters/src/main/java/com/linkedin/android/litr/filter/video/gl/SolidBackgroundColorFilter.java

```diff
@@ -0,0 +1,53 @@
+package com.linkedin.android.litr.filter.video.gl;
+
+import android.graphics.Color;
+import android.opengl.GLES20;
+
+import androidx.annotation.ColorInt;
+import androidx.annotation.FloatRange;
+import androidx.annotation.NonNull;
+
+import com.linkedin.android.litr.filter.GlFilter;
+
+/**
+ * Filter to set video background to solid color
+ */
+public class SolidBackgroundColorFilter implements GlFilter {
+
+    private final float red;
+    private final float green;
+    private final float blue;
+
+    /**
+     * Create using Android integer RGBA color format
+     * @param color color value
+     */
+    public SolidBackgroundColorFilter(@ColorInt int color) {
+        this.red = Color.red(color);
+        this.green = Color.green(color);
+        this.blue = Color.blue(color);
+    }
+
+    /**
+     * Create using OpenGL color format
+     * @param red red value
+     * @param green green value
+     * @param blue blue value
+     */
+    public SolidBackgroundColorFilter(@FloatRange(from = 0, to = 1) float red,
+                                      @FloatRange(from = 0, to = 1) float green,
+                                      @FloatRange(from = 0, to = 1) float blue) {
```

Comment:
RGBA, Alpha? or later?

Reply:
I couldn't come up with good use case for it. We are clearing canvas with that color, not rendering it. By default, canvas is black, so applying transparent background on top of black background will only darken it, which we can do ourselves by passing in lower intensity color values. And we don't have a use case for producing transparent video.
We can always add alpha channel support later.